app/controllers/sessions_controller.rb

```diff
@@ -10,6 +10,10 @@ def create
     end
   end
 
+  def destroy
+    session.destroy
+    redirect_to root_path
+  end
   private
```

Comment:
awesome! thanks for implementing this logout functionality, it's all clean and straightforward!